exercises/chapter12/spago.dhall

```diff
@@ -10,12 +10,10 @@ You can edit this file as you like.
   , "effect"
   , "foldable-traversable"
   , "integers"
-  , "lists"
```

Comment:
Was this removal intentional?

Reply:
Non-intentional, I must've forgotten to add it back in. It was one of those packages that are not used, but still imported potentially for the reader's convenience when solving exercises.